src/module/amplifier.rs

```diff
@@ -47,3 +48,7 @@ impl Module for Amplifier {
         1
     }
 }
+
+pub fn depth(value: f32, depth: f32) -> f32 {
+    1.0 - value + depth * value
```

Comment:
I don't think this is correct. Consider a mod input of 0.5 with a mod depth of 0.2. This will return 0.6 but it should return 0.9 I think?

How about:

```suggestion
    1.0 - depth + depth * value
```

Reply:
ah you're right! incorrectly copied this one over